lib/array.js

```diff
@@ -112,12 +112,30 @@ function uniqueFromArray (array) {
   return [...new Set(array)]
 }
 
+/**
+ * Tests whether an array or string includes a value or not,
+ * and returns either true or false
+ *
+ * @example
+ * arrayOrStringIncludes(['Orange', 'Banana', 'Apple'], 'Orange')
+ * // true
+ *
+ *
+ * @param {(Array|string)} array - Array or string to search
+ * @param {object} searchElement - value to search for
+ * @returns {boolean} Returns true or false
+ */
+function arrayOrStringIncludes (array, searchElement) {
+  return array.includes(searchElement)
+}
+
 module.exports = {
   formatList,
   isArray,
   rejectFromArray,
   selectFromArray,
-  uniqueFromArray
+  uniqueFromArray,
+  arrayOrStringIncludes
```

Comment:
This is the first filter we’ve added that exposes a JavaScript method to Nunjucks templates. As this performs the same role as the native `includes` method, do we need to export it?

Reply:
I'm not totally sure, but I think it probably needs to be exported so that it can be tested?

I don’t think it does any harm exporting it, as whilst it does just delegate to the native javascript method, exporting it means that it's a global function, rather than a function of the Array and String prototypes, so it shouldn’t conflict?

That said, I did end up deciding to name it `arrayOrStringIncludes` rather than just `includes`, just to be safe...

Reply:
Ah, testing, of course!